src/content/getusermedia/resolution/js/main.js

```diff
@@ -103,7 +103,8 @@ video.onplay = function() {
 function getMedia(constraints) {
   if (stream) {
     video.src = null;
-    stream.stop();
+    stream.getAudioTracks().forEach(function(track) { track.stop(); });
```

Comment:
@juberti, filed https://github.com/webrtc/adapter/issues/58 for adding a helper function for stopping all tracks, all video and all audio tracks.